SuperBuild/External_python.cmake

```diff
@@ -153,6 +153,7 @@ if((NOT DEFINED PYTHON_INCLUDE_DIR
     SOURCE_DIR ${EP_SOURCE_DIR}
     BINARY_DIR ${EP_BINARY_DIR}
     CMAKE_CACHE_ARGS
+      -DCMAKE_POLICY_VERSION_MINIMUM:STRING=3.16.3
```

Comment:
```suggestion
```

Minimum required version is `3.20.6`, see https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/blob/master/CMakeLists.txt#L1C31-L1C38

Reply:
@jcfr Should we use 3.20.6 universally as the minimum required for all references in the Slicer/Superbuild/* files?  I think it would be more clear what the minimum build requirements are.